spec/package-manager-spec.coffee

```diff
@@ -286,6 +290,34 @@ describe "PackageManager", ->
             expect(addErrorHandler.callCount).toBe 1
             expect(addErrorHandler.argsForCall[0][0].message).toContain("Failed to load the package-with-invalid-settings package settings")
 
+    describe "when the package metadata includes `activationHooks`", ->
+      [mainModule, promise] = []
+
+      beforeEach ->
+        jasmine.attachToDOM(workspaceElement)
```

Comment:
Seems like we shouldn't need the workspace element to be on the DOM here. Is this a leftover?